com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/SurfaceOptionUIBlock.cs

```diff
@@ -402,12 +442,18 @@ void DrawSurfaceOptionGUI()
         /// <summary>Returns false if there are multiple materials selected and they have different default values for propName</summary>
         float GetShaderDefaultFloatValue(string propName)
         {
+            if (!materials[0].HasProperty(propName))
+                return 0;
+
             // It's okay to ignore all other materials here because if the material editor is displayed, the shader is the same for all materials
             var shader = materials[0].shader;
             return shader.GetPropertyDefaultFloatValue(shader.FindPropertyIndex(propName));
         }
 
-        void DrawAlphaCutoffGUI()
+        /// <summary>
+        /// Draw the Alpha Cutoff GUI (enable, treshold, shadow treshold, prepass cutoff, postpass cutoff)
```

Comment:
Draws the Alpha Cutoff GUI.

I'm not sure whether we should include the list of properties. What is the likelihood that they will change? (for any of these functions).

Reply:
`Enable` and `Threshold` will always appear, for the rest, it depends on the settings of the shader

Reply:
I've removed the list for now